src/screens/CreatePost/index.js

```diff
@@ -366,7 +362,6 @@ const CreatePost = () => {
       );
     }
   };
-
   const takePhoto = async () => {
     const {success, message} = await requestCameraPermission();
     if (success) {
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The code has been indented. This does not affect the functionality of the code.
```diff
+        );
+      }
+    };
+    const takePhoto = async () => {
+      const {success, message} = await requestCameraPermission();
+      if (success) {
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->